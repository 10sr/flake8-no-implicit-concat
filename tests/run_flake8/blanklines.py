a = ["aaa"



     "bbb"]
b = ["aaa"  # comment



     "bbb"]

def c():
    a = ["aaa"  # comment


         "bbb"]
    return a

print(a)
print(b)
print(c())
