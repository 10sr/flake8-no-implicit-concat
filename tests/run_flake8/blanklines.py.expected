./blanklines.py:1:11: NIC002 Implicitly concatenated str literals over multiple lines
./blanklines.py:6:11: NIC002 Implicitly concatenated str literals over multiple lines
./blanklines.py:13:15: NIC002 Implicitly concatenated str literals over multiple lines
